lib/tasks/build.rake

```diff
@@ -17,15 +17,34 @@ task build: :environment do
     require 'render_anywhere'
     include RenderAnywhere
 
-    FileUtils.mkdir('dist') unless File.directory?('dist')
-    FileUtils.rm_rf dist_folder
-    FileUtils.mkdir dist_folder
+    prep_dist
+
     zip rails_asset_path, dist_assets_path
     File.write(dist_cascade_block_path, render(file: 'layouts/cascade-assets.xml', layout: false))
     FileUtils.cp instructions, dist_folder
   end
 end
 
+task build_omni_nav: :environment do
```

Comment:
Almost overlooked a couple things:

- Include a description here for `rake -T`. 
- Also, don't forget to update the README to include this and explain why it was created.